proposals/lip-0066.md

```diff
@@ -45,6 +45,14 @@ We define the following constants:
 | `PrivateKeyEd25519` | bytes  | Byte sequences of length `ED25519_PRIVATE_KEY_LENGTH`. | An Ed25519 private key. |
 | `PrivateKeyBLS`     | bytes  | Byte sequences of length `BLS_PRIVATE_KEY_LENGTH`. | An BLS private key. |
 
+## uint32be Function
+We define the function `uint32be` to convert bytes to unsigned integer values.
+
+```python
+def uint32be(chainID: bytes) -> int:
```

Comment:
minor comment, but in other LIPs we use the type `uint32`, so we could use that in the output type instead of `int`

Reply:
Updated.